docs/guides/code_push_release_app_store.md

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position: 3
+title: 🍎 Release a Code Push App (iOS)
+description: Release a Code Push App (iOS)
+---
+
+# Releasing a code push app to the Apple App Store
+
+This guide walks through releasing a code push app to the Apple App Store and applying a patch to that release.
+
+The app we will be releasing in this guide is [`Shorebird Clock`](TODO), our demo code push app. ([source](https://github.com/shorebirdtech/time_shift/))
+
+## Prequisites
+
+To follow along with this guide, you will need the following:
+
+1. An existing Shorebird app. If you don't have one, you can create one by following the [code push quickstart](./code_push_quickstart/) guide.
+2. Access to hardware running macOS. This is required to build iOS apps.
+3. A valid Apple Developer account. This is required to release iOS apps.
+4. An app in [App Store Connect](https://appstoreconnect.apple.com/). See the [official documentation](https://developer.apple.com/help/app-store-connect/create-an-app-record/add-a-new-app) for more information about how to create one.
+5. The [Transporter app](https://apps.apple.com/us/app/transporter/id1450874784). We will be using this to upload our app to the App Store.
+
+## Creating a release
+
+### Determine the release version
+
+Run `shorebird releases list` to see the current set of releases:
+
+```
+⑆ shorebird releases list
+🚀 Releases (f2184ee6-9a85-498c-bfeb-114d638c462e)
+┌─────────┬──────┐
+│ Version │ Name │
+├─────────┼──────┤
+│ 1.0.3+1 │ --   │
+└─────────┴──────┘
+```
+
+This shows that the most recent release is `1.0.3+1`. Our next version will be `1.0.4+1`.
+
+### Create a release in App Store Connect
+
+Because the App Store does not include the build number (the `+1` part of `1.0.4+1`) in app versions, this will show up in the App Store as `1.0.4`. Follow the instructions in the [official documentation](https://developer.apple.com/help/app-store-connect/update-your-app/create-a-new-version) to create a new version in App Store Connect.
```

Comment:
App Store does make apps unique by build number, just hides it from the user, right?

Reply:
Correct. The developer sees the build numbers, but build numbers are not included in app versions the way they are in the Play Store.